public/src/managers/CourseManager.ts

```diff
@@ -378,10 +387,11 @@ export class CourseManager {
         if (assignments.length > 0) {
             const submissions =
                 await this.courseProvider.getAllGroupLabInfos(groupCourse.course.getId(), group.getId());
-
             for (const a of assignments) {
-                const submission = submissions.find((sub) => sub.assignmentid === a.getId());
-                groupCourse.assignments.push({ assignment: a, latest: submission, authorName: group.getName() });
+                if (a.getIsgrouplab()) {
```

Comment:
Should be `isGroupLab()`, unless this is generated code.

Reply:
It is a protobuf-defined class, you can only use getters here.